### PR TITLE
Corona news site

### PR DESCRIPTION
**Pull request by MayaBlixt**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  padding: 3rem;
+  width:300px;
+  height:90%; 
+}
+
+.news_cards {
+	max-width: 1000px;
+	margin: 0 auto;
+	display: grid;
+	grid-gap: 3rem;
```
Comment (EmOlsz) [inline style.css:76]: Great job with grid approach, but your website just after entering desktop sizes has horizontal scroll - row of cards is wider than the browser window itself

File: style.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+		justify-content: space-around;	
+	}
+	header h1{
+		display:none;
+	}
+	header h2{
+		display:none;
+	}
+	.big-news{
+		padding: 20px 5px 20 px 5px;
```
Comment (EmOlsz) [inline style.css:106]: `20 px`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+		color:black;
+		text-decoration:none;
+		width:50%;
+		display:flex;
+		justify-content: space-around;
+	}
+	header h1{
+		font-size: 40px;
+		margin:0px 30px 0px 10px;
+		padding:0px;
```
Comment (EmOlsz) [inline style.css:138]: 0px

File: style.css [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+		padding: 1rem;
+		width:650px;
+		height:90%;	
+	  }	
+	  html { font-size: 18px; 
+	  }
+	  
+}
+/* shake images*/
+@keyframes shake {
```
Comment (EmOlsz) [inline style.css:162]: Super nice animation :)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+	width: 220px;
+	padding:10px;
+  } 
+   
+  .card img {
+	border: 10px solid #fff;
+	border-bottom: 45px solid #fff;
+	-webkit-box-shadow: 3px 3px 3px #777;
+	   -moz-box-shadow: 3px 3px 3px #777;
+			box-shadow: 3px 3px 3px #777;
```
Comment (EmOlsz) [inline style.css:192]: It's a great plus that you remembered about other browsers!
But what happened with indents over here?

File: style.css [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+  }
+   
+  .card p {
+	position: absolute;
+	text-align: center;
+	width: 100%;
+	bottom: 0px;
```
Comment (EmOlsz) [inline style.css:199]: 0px

**Review by EmOlsz — CHANGES_REQUESTED** [review]
That's a very impressive project Marjaana! I have to say you suprised in couple moments with how big of a research you did regarding some technologies :) On the other hand I need to say that some parts unfortunately needs to be fixed. Your Grid in main section is not working properly, desktop view just after you jump from tablet view is not responsive and horizontal scroll is present... I will need to ask you to work on that again. Maybe flexbox will be better solution since it's required by us and css-grid not?

Apart from that, here are the things you did great and you can continue on doing so :
1. Nice animation, both from technical and design point of view
2. Custom google fonts imported
3. Favicons for many types of devices
4. Webmanifest - even though it's unnecessary over here, I have to tell you shocked me in a positive way :)
5. Meaningful comments both inside HTML and CSS file

Things you need to work on :
1. Indents - your code looks a bit chaotic and unaesthetic because of them
2. Responsivenes - as said above, I will need to ask you for fixed before I can approve your code

File: index.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+      <a href="https://www.worldometers.info/coronavirus/?">Corona cases</a>
+      <a href="https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/bekraftade-fall-i-sverige/">FHM</a>
+      <img class= "nav_icon" src ="./assets/icon-corona-black.png" alt="Corona virus">
+      <a href="https://www.youtube.com/watch?v=bqx19f0ceUo">Corona TikTok Dance</a>
+      <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">WHO</a>
+  </div>
+</header>
+  <!-- The big news content -->
+    <section class="big-news">
+        <div class="news"><img src="./assets/corona_3.jpg" alt="Corona virus"></div>
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply index.html:42]: Fixed indentation

File: index.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="news"><p>According to the CDC, there is no specific antiviral treatment recommended for infection with novel coronavirus. Anyone infected with the virus should receive supportive care to help relieve symptoms.The CDC recommends the following prevention measures:
+          Wash your hands often with soap and water for at least 20 seconds. 
+          </p></div>
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply index.html:45]: Fixed line break

File: index.html [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+    <!-- News card grid-->
+    <div class="news_cards">
+      <div class="card"><img src="./assets/corona_1.jpg" alt="Stay at home"><p>Corona from home</psta></div>
+      <div class="card"><img src="./assets/corona_2.jpg" alt="Corona game"><p>Games at home></p></div>
+      <div class="card"><img src="./assets/corona_bignews.jpg" alt="Social distancing"><p>Keep your distance</p></div>
+      <div class="card"><img src="./assets/corona_4.jpg" alt="Social distancing"><p> Social distance</p></div>
+      <div class="card"><img src="./assets/corona_5.jpg" alt="Mask and hand alcohol"><p> Will mask do it?</p></div>
+      <div class="card"><img src="./assets/corona_6.jpg" alt="Mask"><p>Preventing Corona</p></div>
+      <div class="card"><img src="./assets/corona_7.jpg" alt="Coronavirus"><p>Origin of Corona </p></div>
+      <div class="card"><img src="./assets/corona_8.jpg" alt="Pills"><p>Corona vitamins</p></div>
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply index.html:57]: Fixed. img and p elements are in separate lines

File: index.html [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="instagram">
+        <a href="https://www.instagram.com/marjaana_star/?hl=en" target="_blank"><img src="./assets/instagram-icon.png" alt="Click to visit us on Instagram"></a>
+      </div>
+      <div class="github">
+        <a href="https://github.com/MayaBlixt" target="_blank"><img src="./assets/github-9-64.png" alt="Click to visit us on Github"></a>
+      </div>
+      <div class="linkedin">
+        <a href="https://www.linkedin.com/in/marjaana-tienaho-0576b915/" target="_blank"><img src="./assets/linkedin_icon.png" alt="Click to visit me on Linkedin"></a>
+      </div>
+      </footer>
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply index.html:71]: Fixed, indentation.

File: site.webmanifest [hunk 11]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply site.webmanifest:1]: Removed webmanifest.

File: style.css [hunk 12]
```diff
@@ -0,0 +1,219 @@
+
+body{
+  color:white;
+  margin:0px;
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply style.css:4]: Fixed. Updated margin to 0.

File: style.css [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+  font-family: 'Baloo Tamma 2', cursive;
+  background-image: url('./assets/texture_08_by_oleo_kun_d1a4a8l.jpg');
+}
+html { font-size: 13px; 
+}
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply style.css:9]: Fixed, indentation.

File: style.css [hunk 14]
```diff
@@ -0,0 +1,219 @@
...
+	padding: 30px;
+	background-color: whitesmoke;
+	height: 70px;
+	display: flex;
+	align-items: center;
+}
+header h1{
+	font-size: 40px;
+	margin:0px 30px 0px 10px;
+	padding:0px;
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply style.css:21]: Fixed. Changed to 0.

File: style.css [hunk 15]
```diff
@@ -0,0 +1,219 @@
...
+	color:black;
+}
+
+header a{
```
[the remark this answers is not in the document]
Comment (MayaBlixt) [reply style.css:25]: Fixed.

File: style.css (hunk 2 above)
Comment (MayaBlixt) [reply style.css:106]: Fixed.

File: style.css (hunk 3 above)
Comment (MayaBlixt) [reply style.css:138]: Fixed

File: style.css (hunk 5 above)
Comment (MayaBlixt) [reply style.css:192]: Fixed, indents.

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**